adding izzi's activity

iHinks opened the pull request.

iHinks committed:
added izzi activity

wenzeslaus commented on hinks.py:
```diff
@@ -0,0 +1,70 @@
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
```
The import statement linting tool _isort_ wants you to include an extra empty line after the imports. (There should be two empty lines between the imports and the function definitions.) The relevant error from the Super-Linter check (which includes the isort tool) is:

```
ERROR: /github/workspace/hinks.py Imports are incorrectly sorted and/or formatted.
--- /github/workspace/hinks.py:before	2021-02-11 15:53:16.684052
+++ /github/workspace/hinks.py:after	2021-02-11 15:54:03.956293
@@ -16,6 +16,7 @@
 """
 
 import grass.script as gs
+
 
 def run_ponds(scanned_elev, env, **kwargs):
     repeat = 20
```

wenzeslaus commented on hinks.py:
```diff
@@ -0,0 +1,70 @@
+    # filter depression deeper than 0.1 m to
+    gs.mapcalc('{new} = if({out} - {scan} > 0.1, {out} - {scan}, null())'.format(new='ponds', out=output,
+                                                                                 scan=scanned_elev), env=env)
+    gs.write_command('r.colors', map='ponds', rules='-', stdin='0% aqua\n100% blue', env=env)
+# this part is for testing without TL
+
+def run_waterflow(scanned_elev, env, **kwargs):
```
There should be two empty lines in between functions according to the PEP8 code style standard for Python, so that's what the Flake8 linting tool wants you to do here and before `def main():` (the first error is the same as from the import linter). The relevant output from Super-Linter (which includes Flake8) is here:

```
2021-02-11 15:54:01 [ERROR]   Found errors in [flake8] linter!
2021-02-11 15:54:01 [ERROR]   Error code: 1. Command output:
------
/github/workspace/hinks.py:20:1: E302 expected 2 blank lines, found 1
/github/workspace/hinks.py:33:1: E302 expected 2 blank lines, found 1
/github/workspace/hinks.py:39:1: E302 expected 2 blank lines, found 1
```

wenzeslaus commented on hinks.py:
```diff
@@ -0,0 +1,70 @@
+    gs.run_command("r.resamp.stats", input=elevation, output=elev_resampled, env=env)
+    gs.run_command("g.copy", raster=[elev_resampled, "scan_saved"], env=env)
+
+    # create points
+    points = "points"
+    gs.write_command(
+        "v.in.ascii",
+        flags="t",
+        input="-",
+        output=points,
+        separator="comma",
+        stdin="638432,220382\n638621,220607",
+        env=env,
+    )
```
If you are not using points, it is best to remove the code. (The code is still in the examples if you need it in the future as well as Git history in this case.) No code, no confusion, no errors as we can see from these relevant [Murphy's laws](http://www.murphys-laws.com/murphy/murphy-computer.html):

> * Every non-trivial program contains at least one bug.
> * Every non-trivial program can be simplified by at least one line of code.
> * Conclusion: Every non-trivial program can be simplified to one line of code, and it will contain a bug.

wenzeslaus commented on hinks.py:
```diff
@@ -0,0 +1,70 @@
+        stdin="638432,220382\n638621,220607",
+        env=env,
+    )
+    # run_ponds(scanned_elev=elev_resampled, env=env, points=points)
```
If you define the function, you should also run it here to test it.

wenzeslaus reviewed (changes requested):
This is looks pretty good, although it has some major issues which prevents it from running and a couple of formal issues (see below).

The files are in the root (main) directory, but they need to be in the `activities` directory in order to be considered for processing (picked by the automated checks and in Tangible Landscape). You can move them, e.g., using:

```
git mv config_hinks.json activities
git mv hinks.json activities
```

iHinks committed:
moved to activities folder

iHinks committed:
moved to activities folder

also added two empty lines where necessary and removed "points = "points"" and "output=points,"
